### ISSUE TITLE
Question 

### ISSUE THREAD

**Issue by Davibiyuu** [Davibiyuu/zashitacoil#2]
I have my own small company, most of the employees work remotely. Lately I see how their productivity decreases, I do not know what to do about it. I came across an article about [https://www.kickidler.com/info/best-project-management-software.html](https://www.kickidler.com/info/best-project-management-software.html)  the best programs for remote monitoring, what do you think, will it work? Not to cheat, right?

**Comment by aadilraza077** [comment]
Yes its works also check out other monitoring software compare and choose the best such as [Timechamp](https://timechamp.io/) and so on
